dxr/app.py

```diff
@@ -307,6 +307,14 @@ def browse(tree, path=''):
                             frozen['generated_date'])
 
 
+def concat_plugin_headers(plugin_list):
+    """Return a list of the concatenation of all browse_headers in the
+    FolderToIndexes of given plugin list.
+    """
```

Comment:
For consistency with the rest of the code, there should be a skipped blank line before the closing """.